plugin/session_buffer.py

```diff
@@ -265,6 +273,32 @@ def some_view(self) -> Optional[sublime.View]:
             return sv.view
         return None
 
+    def _if_view_version_stable(self, f: Callable[[sublime.View, Any], None], version: int) -> Callable[[Any], None]:
```

Comment:
`_view_remained_unchanged`
rename if you find the suggested name more suitable

Reply:
I think it reads better at use-site with "if" but I've adapted part of your suggestion.